### ISSUE TITLE
Expose system ml algorithms.

### ISSUE THREAD

**Issue by holdenk** [sparklingpandas/sparklingml#7]

**Comment by j143-zz** [comment]
Is this feature related to using the [SystemML algorithms](http://github.com/apache/systemml/tree/master/scripts/algorithms/GLM.dml), in here.(?)

**Comment by holdenk** [comment]
Yes, the idea would be to generate wrappers for them so they could be used in Spark ML pipelines.
